### ISSUE TITLE
Following a user does not download their messages

### ISSUE THREAD

**Issue by derekchiang** [ssbc/scuttlebot#523]
I have a setup where I have two nodes A and B and a pub server P.  A and B are both connected with the pub.  A and B [followed each other](https://scuttlebot.io/docs/social/follow-users.html) and it worked according to `sbot friends.get`.

However, A is simply not getting B's messages.  On the other hand, B is getting A's messages.

Is this a bug or am I missing something?

**Comment by dominictarr** [comment]
is the pub following A and B, or at least one of them? is this a script, can you post your code?

**Comment by derekchiang** [comment]
Yeah the hub is following both A and B, since they both accepted the hub's
invites. However, I've found that after I restart A and B, they start
receiving messages from one another.  Is that the expected behavior?

I don't quite have a script yet since this is part of a larger deployment.

**Comment by christianbundy** [comment]
No, having to restart for gossip to work shouldn't be expected behavior.

Could you post your Scuttlebot version and your relevant config changes (if any)?

**Comment by mmckegg** [comment]
I've seen this recently while testing patchwork onboarding.

Accepting the pub invite brought in all of the posts from the pub's feed, but only some of the feeds it followed. This happened reliably every time I tried (same pub though, so maybe it was crashing?)  

Had to restart sbot, or reconnect to that pub before the rest of the messages would sync.
